run-tests.py

```diff
@@ -11,16 +11,10 @@
 PROJECT_DIR = os.path.dirname(__file__)
 subprocess.call(['find', PROJECT_DIR, '-name', '*.pyc', '-delete'])
 
-# Check for style
-config_file = os.path.join(PROJECT_DIR, 'flake8.cfg')
 # These paths should all pass PEP-8 checks
 paths_to_check = [
     'server/pulp/server/tasks/',
     'server/test/unit/server/']
-paths_to_check = [os.path.join(PROJECT_DIR, p) for p in paths_to_check]
```

Comment:
By dropping this, you cause this script to depend on CWD when it was CWD agnostic before. I think it's important to be able to call the tests with any path from any working directory. If we simply require this param, expect the caller to pass it as absolute paths, and keep this line, everything will continue to work.